pkg/disk/v1alpha2/disk.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1alpha2
 
 import (
-	ndmapisv1alpha1 "github.com/openebs/maya/pkg/apis/openebs.io/ndm/v1alpha1"
+	ndmapis "github.com/openebs/maya/pkg/apis/openebs.io/ndm/v1alpha1"
```

Comment:
What if we do not change this file ? Are we using this `pkg/disk/v1alpha2` elsewhere for block device requirement?

Reply:
same as above